### DISCUSSION TITLE
TouchableOpcaity setComponentTheme borderRadius not working?

### DISCUSSION THREAD

**Discussion by wortkotze** [wix/react-native-ui-lib#1781] [Q&A]
Hi, 

I try to uset the setComponentTheme to apply our Designsystem to the existing component. It is working on a normal View component but the properties borderRadius and padding are not working on the Touchable Opacity. The Backgorund Color is working. Is this a known issue or do I do something wrong :(

thx for the help
Stephan

ThemeManager.setComponentTheme("TouchableOpacity", (props, context) => {
  return {
    borderRadius: props.Background ? 4 : 0,
    backgroundColor: props.Background
      ? "rgba(255,255,255,0.25)"
      : "transparent",
    padding: props.Background ? 10 : 0,
  };
});

**Comment by ethanshar** [comment]
Hi @wortkotze 
Please consider reporting a bug instead of a discussion. 

Thank you 🙏
